0.neurovault_images.py

```diff
@@ -11,14 +13,24 @@
 import pandas
 import os
 import sys
+import nibabel
+
+try:
+    __IPYTHON__
+    base='/Users/poldrack/data_unsynced/forward-modeling-cognitive-concepts'
```

Comment:
ok, this fix is pushed, let me know if I need to do another PR

> On Aug 14, 2016, at 5:02 PM, Vanessa Sochat <notifications@github.com> wrote:
> 
> In 0.neurovault_images.py <https://github.com/vsoch/forward-modeling-cognitive-concepts/pull/2#discussion_r74711440>:
> 
> > @@ -11,14 +13,24 @@
> >  import pandas
> >  import os
> >  import sys
> > +import nibabel
> > +
> > +try:
> > +    __IPYTHON__
> > +    base='/Users/poldrack/data_unsynced/forward-modeling-cognitive-concepts'
> Do you want to hard code your system path into the script?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/vsoch/forward-modeling-cognitive-concepts/pull/2/files/bdc3db014bb069083d6ed82b76323b898ae61c90#r74711440>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA1KkFc-EMJcbQ7NEMgrc9H-9UiHi7yWks5qf6yvgaJpZM4JkA5e>.
> 

—
Russell A. Poldrack
Albert Ray Lang Professor of Psychology
Bldg. 420, Jordan Hall
Stanford University
Stanford, CA 94305

poldrack@stanford.edu
http://www.poldracklab.org/